src/test/java/se/sundsvall/businessrules/rule/impl/parkingpermit/criteria/ExpiringParkingPermitCriteriaTest.java

```diff
@@ -0,0 +1,173 @@
+package se.sundsvall.businessrules.rule.impl.parkingpermit.criteria;
+
+import static generated.se.sundsvall.citizenassets.Status.ACTIVE;
+import static generated.se.sundsvall.citizenassets.Status.EXPIRED;
+import static java.util.Collections.emptyList;
+import static java.util.UUID.randomUUID;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import static se.sundsvall.businessrules.integration.citizenassets.CitizenAssetsClient.PARTY_ID_PARAMETER;
+import static se.sundsvall.businessrules.integration.citizenassets.CitizenAssetsClient.TYPE_PARAMETER;
+import static se.sundsvall.businessrules.rule.impl.parkingpermit.criteria.ExpiringParkingPermitCriteria.EXPIRATION_PERIOD_IN_MONTHS;
```

Comment:
See comment in ExpiringParkingPermitCriteria class

Reply:
Done